falco/README.md

```diff
@@ -49,7 +53,13 @@ Falco needs a **driver** (the [kernel module](https://falco.org/docs/event-sourc
 
 By default the drivers are managed using an *init container* which includes a script (`falco-driver-loader`) that either tries to build the driver on-the-fly or downloads a prebuilt driver as a fallback. Usually, no action is required.
 
-If a prebuilt driver is not available for your distribution/kernel, Falco needs **kernel headers** installed on the host as a prerequisite to build the driver on the fly correctly. You can find instructions for installing the kernel headers for your system under the [Install section](https://falco.org/docs/getting-started/installation/) of the official documentation.
+Kernel versions and flavors are automatically discovered by the kernel-crawler. At the time being, it runs weekly, on each Monday mornings. We have a site where users can check for the discovered kernel flavors and version, [example for Amazon Linux 2](https://falcosecurity.github.io/kernel-crawler/?arch=x86_64&target=AmazonLinux2).
```

Comment:
```suggestion

##### Pre-built drivers

The [kernel-crawler](https://github.com/falcosecurity/kernel-crawler) automatically discovers kernel versions and flavors. At the time being, it runs weekly. We have a site where users can check for the discovered kernel flavors and versions, [example for Amazon Linux 2](https://falcosecurity.github.io/kernel-crawler/?arch=x86_64&target=AmazonLinux2).
```
Added subsection.
Removed _on Monday_, since that may change at any time without notice.